### ISSUE TITLE
Properly split icon bundles

### ISSUE THREAD

**Issue by CarloPalinckx** [MyOnlineStore/bricks#307]
### Is your feature request related to a problem? Please describe ⚡️
Right now we are shipping all icon svg's once an Icon is used. We can split these up so that users end up just using the required icon only.

### Describe the solution you'd like ✨
We could create a script that codegens a component for every icon in the assets folder. Ideally they would be imported as something along these lines:
```ts
import Camera from 'react-feather/dist/icons/camera';

class MyClass extends React.Component {
  render() {
    return <Camera />
  }
}
```

**Comment by woutsluiter** [comment]
Would the icon components still have a `size` and `color` prop? `color` would be easy to do I guess, but the `size` prop might be difficult to type, seeing as how not every icon is available in every size. Or we could just structure them in folders.

**Comment by hborghols** [comment]
Oui s'il vous plaît.

**Comment by CarloPalinckx** [comment]
Resolved in #329
